ios/Lunes.xcodeproj/project.pbxproj

```diff
@@ -785,12 +785,12 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
-				CURRENT_PROJECT_VERSION = 1;
+				CURRENT_PROJECT_VERSION = 2;
```

Comment:
Not sure if we should commit these changes, I think we don't do for Integreat as those will be outdated all the time.

Reply:
oke, i will have a closer look at this, when doing the ios-ci-stuff.